schema/ethernet.yml

```diff
@@ -168,6 +168,14 @@ properties:
         type: integer
         minimum: 1
         maximum: 4094
+      mac-address-bypass:
+        description: Trigger mac-auth when a new ARP is learned.
+        type: boolean
+      mac-address-bypass-timout:
```

Comment:
What is the unit, seconds? milliseconds?

Reply:
Also, i think there's a typo?
>      mac-address-bypass-timout
..
>      mac-address-bypass-tim**e**out

Reply:
mac-address-bypass:
        description: Enables bypass when a device does not support 802.1X authentication (e.g., printers, IP phones)
        type: boolean
      **mac-address-bypass-timeout:**
        description: Defines the time period **(in minutes)** for which a MAC address is allowed access to the network without requiring reauthentication, after being authenticated or allowed via MAC Authentication Bypass (MAB).
        type: integer